kncaputo/whats-cookin
# Feature/scripts

## What's this PR do?
- Add scripts functions on load functionality, saving to recipes My Favorites, saving to What's Cookin', and for the Show Recipe modal.

## Where should the reviewer start?
- Start by reviewing scripts and corresponding CSS. 

## How should this be manually tested?
- After running `open src/index.js`, click around on the site, adding recipes to favorites and what's cookin. Navigate to those pages to see the results. On a recipe card, click `show recipe` button to see a modal with instructions.

## Any background context you want to provide?
- Known issues to fix:
  1. Nav bar links change color based on what page the user is viewing, but they are not always correct.
  2. Need to style recipe modal.
  3. Need to add color heart image for liked recipes.

## Screenshots/GIFs
<img width="1512" alt="Screen Shot 2020-10-18 at 10 34 16 AM" src="https://user-images.githubusercontent.com/68157759/96371224-a5a2a680-112e-11eb-8be0-00e809197597.png">
<img width="1506" alt="Screen Shot 2020-10-18 at 10 34 43 AM" src="https://user-images.githubusercontent.com/68157759/96371227-a89d9700-112e-11eb-98d6-30d9e0296a0d.png">
<img width="1511" alt="Screen Shot 2020-10-18 at 10 43 07 AM" src="https://user-images.githubusercontent.com/68157759/96371270-d7b40880-112e-11eb-8865-4c78e7e79168.png">
